webpack.config.js

```diff
@@ -1,96 +1,109 @@
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const {CleanWebpackPlugin} = require('clean-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
 
 const commonConfig = {
   devtool: 'eval-cheap-module-source-map',
   resolve: {
     extensions: ['.js', '.jsx'],
+    alias: {
+      '@ml': path.resolve(__dirname, 'src')
+    }
```

Comment:
this is the only actual change in this file -- everything else (minus the new `require` statement on line 3) is prettier formatting